bindings/python/vara-feature/pybind_Feature.cpp

```diff
@@ -55,42 +47,37 @@ void init_feature_module_feature(py::module &M) {
             return py::make_iterator(F.excludes_begin(), F.excludes_end());
           },
           py::keep_alive<0, 1>(), R"pbdoc(Excluded features)pbdoc")
-      .def(
-          "add_exclude", &vf::Feature::addExclude,
-          R"pbdoc(Exclude a Feature, i.e., it can not be pressent when this Feature is present.)pbdoc")
       .def("is_excluded", &vf::Feature::isExcluded,
            R"pbdoc(Checks if a Feature is excluded by this Feature.)pbdoc")
       //===----------------------------------------------------------------===//
       // Implications
       .def(
           "implications",
           [](vf::Feature &F) {
-            return py::make_iterator(F.excludes_begin(), F.excludes_end());
+            return py::make_iterator(F.implications_begin(),
+                                     F.implications_end());
           },
           py::keep_alive<0, 1>(), R"pbdoc(Implicated features)pbdoc")
-      .def(
-          "add_implication", &vf::Feature::addImplication,
-          R"pbdoc(Implicate another Feature, i.e., selecting this feature implies the otherone.)pbdoc")
-      .def("implicates", &vf::Feature::implies,
+      .def("is_implied", &vf::Feature::isImplied,
```

Comment:
```suggestion
      .def("is_implied_by", &vf::Feature::isImplied,
```
This slight name change should make it a bit clearer